app/medInria/medRegistrationWorkspace.cpp

```diff
@@ -45,31 +45,14 @@ class medRegistrationWorkspacePrivate
 
 medRegistrationWorkspace::medRegistrationWorkspace(QWidget *parent) : medAbstractWorkspace(parent), d(new medRegistrationWorkspacePrivate)
```

Comment:
Not for registration workspace. It has huge particularities which do not allowed to change it to a `medSelectorWorkspace` with `medAbstractSelectableToolBox` for now. I spoke about that with flo yesterday (or the day before, do not remember).

But i created a medSelectorWorkspace, as you can see in the other workspaces (excepted Visu and Pipelines).